lms/static/js/learner_dashboard/views/program_header_view_2017.js

```diff
@@ -0,0 +1,57 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'text!../../../templates/learner_dashboard/program_header_view_2017.underscore',
+        'text!/static/images/programs/micromasters-program-details.svg',
+        'text!/static/images/programs/xseries-program-details.svg',
+        'text!/static/images/programs/professional-certificate-program-details.svg'
+    ],
+         function(Backbone, $, HtmlUtils, pageTpl, MicroMastersLogo,
+                  XSeriesLogo, ProfessionalCertificateLogo) {
+             return Backbone.View.extend({
+                 breakpoints: {
+                     min: {
+                         medium: '768px',
+                         large: '1180px'
+                     }
+                 },
+
+                 el: '.js-program-header',
+
+                 tpl: HtmlUtils.template(pageTpl),
+
+                 initialize: function() {
+                     this.render();
+                 },
+
+                 getLogo: function() {
+                     var logo = false,
+                         type = this.model.get('programData').type;
+
+                     if (type === 'MicroMasters') {
```

Comment:
It's going to be difficult because of the way we are passing in the SVG logos. I suppose we could have a logoCollection, and we could do ```logoCollection.findWhere({type: type})``` so at least we are moving the hard-coded program types into the logo Collection.